gkTally/profitLoss.py

```diff
@@ -1,87 +1,100 @@
-#!/usr/bin/python3
-
 import csv
 import argparse
 from datetime import datetime
 
-date_format = "%m/%d/%Y"
 
-def parse_args():
-    '''
-    Parse command line arguments '''
+def parseArgs():
+    """
+    Parse command line arguments """
```

Comment:
Glad to see the comment. NIT: the closing 3 " should be in next line.

Reply:
agree